README.md

```diff
@@ -1,9 +1,11 @@
-# libsbn
+# bito
 
-[![Docker Repository on Quay](https://quay.io/repository/matsengrp/libsbn/status "Docker Repository on Quay")](https://quay.io/repository/matsengrp/libsbn)
-[![Travis CI status](https://travis-ci.org/phylovi/libsbn.svg?branch=main)](https://travis-ci.org/phylovi/libsbn)
+[![Docker Repository on Quay](https://quay.io/repository/matsengrp/bito/status "Docker Repository on Quay")](https://quay.io/repository/matsengrp/bito)
```

Comment:
We'll need to change these... if we want to continue using Quay then please make an account and send me your login info.